main/src/com/google/refine/operations/column/ColumnAdditionByFetchingURLsOperation.java

```diff
@@ -134,8 +151,34 @@ public ColumnAdditionByFetchingURLsOperation(
         _delay = delay;
         _cacheResponses = cacheResponses;
         _httpHeadersJson = httpHeadersJson;
+
+        List<Header> headers = new ArrayList<Header>();
+        if (_httpHeadersJson != null) {
+            for (HttpHeader header : _httpHeadersJson) {
+                if (header.name != null && header.name != "" && header.value != null && header.value != "") {
```

Comment:
Shouldn't checks like `header.name != ""` use `equals` instead?

Reply:
Yes, of course! I'll fix it.

Reply:
Done.